lib/internal/test_runner/runner.js

```diff
@@ -203,10 +199,15 @@ class FileTest extends Test {
     this.reportStarted();
     this.#handleReportItem(ast);
   }
+  reportStarted() {}
```

Comment:
It seems like this shouldn't be needed.

Reply:
see https://github.com/nodejs/node/pull/46440#discussion_r1096573795